CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # main
 
+- Iterating over ids in a `GridSpace` is now 2 times faster than before.
```

Comment:
Wait, this improves nearest neighbor searches right? The way it is phrased now is missleading; makes you think it improves iterating over `allids(model)`. You can say "Iterating over ids in neighborhood searches (`nearby_ids` and derivative functions)...".

Reply:
ops, sure, thought it was clear but it wasn't!